src/apps/common/submodules/portWizard/portWizard.js

```diff
@@ -1321,35 +1321,60 @@ define(function(require) {
 					.on('click', function(event) {
 						event.preventDefault();
 
-						self.portWizardHelperSavePort(_.merge({}, args, {
-							stopErrorPropagation: true,
-							success: function(requestId) {
+						monster.waterfall([
+							function(waterfallCallback) {
+								self.portWizardHelperSavePort(_.merge({}, args, {
+									stopErrorPropagation: true,
+									success: function(requestId) {
+										waterfallCallback(null, requestId);
+									}
+								}));
+							},
+							function(requestId, waterfallCallback) {
+								// Add request ID to args object, to know that the request
+								// has already been saved at this point
+								args = _.merge({
```

Comment:
You should use `lodash#set` to assign a value to that property as it takes care of creating those nested objects if missing.

Reply:
I used `lodash#merge` to not to modify the original `args` object, but I think that it does not really matter, as it probably is modified along the way anyways. I'll change it to `lodash#set`.